### PR TITLE
Modernize

### PR DESCRIPTION
**Pull request by wagenet**
Run ember-cli-update and get things working.

This will require a new major release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ An addon geared towards making Ember upgrades easier by allowing you to work thr
 
 [Short description of the addon.]
 
+<<<<<<< HEAD
```
Comment (rwjblue) [inline README.md:9]: merge conflict

File: README.md [hunk 2]
```diff
@@ -18,6 +19,12 @@ This addon can help manage deprecations from Ember 1.x to 3.x.
 
 ## Usage
 
+### Compatibility
+
+* Ember.js v3.16 or above
+* Ember CLI v2.13 or above
```
Comment (rwjblue) [inline README.md:25]: Let's update this to say Ember CLI 3.16 also

File: config/release.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /* jshint node:true */
```
Comment (rwjblue) [inline config/release.js:1]: We should delete this file

File: index.js [hunk 4]
```diff
@@ -3,13 +3,13 @@
 module.exports = {
   name: require('./package').name,
 
-  init: function() {
+  init: function () {
```
Comment (rwjblue) [inline index.js:6]:
```suggestion
  init() {
```

File: index.js [hunk 5]
```diff
@@ -3,13 +3,13 @@
...
     this._super.init && this._super.init.apply(this, arguments);
 
     this._templateDeprecations = [];
   },
 
-  _shouldInclude: function() {
+  _shouldInclude: function () {
```
Comment (rwjblue) [inline index.js:12]:
```suggestion
  _shouldInclude() {
```

File: index.js [hunk 6]
```diff
@@ -19,24 +19,29 @@ module.exports = {
     return app.tests;
   },
 
-  included: function() {
+  included: function () {
```
Comment (rwjblue) [inline index.js:22]:
```suggestion
  included() {
```

File: index.js [hunk 7]
```diff
@@ -19,24 +19,29 @@ module.exports = {
...
+        'vendor/ember-cli-deprecation-workflow/deprecation-workflow.js'
+      );
       app.import('vendor/ember-cli-deprecation-workflow/main.js');
     } else {
       app.import('vendor/ember-debug-handlers-polyfill/prod.js');
     }
   },
 
-  treeForVendor: function(tree) {
+  treeForVendor: function (tree) {
```
Comment (rwjblue) [inline index.js:37]:
```suggestion
  treeForVendor(tree) {
```

File: index.js [hunk 8]
```diff
@@ -45,40 +50,46 @@ module.exports = {
...
 
-      destDir: 'ember-cli-deprecation-workflow'
+      destDir: 'ember-cli-deprecation-workflow',
     });
 
     return mergeTrees([tree, configTree], { overwrite: true });
   },
 
-  _findHtmlbarsPreprocessor: function(registry) {
+  _findHtmlbarsPreprocessor: function (registry) {
```
Comment (rwjblue) [inline index.js:59]:
```suggestion
  _findHtmlbarsPreprocessor(registry) {
```

File: index.js [hunk 9]
```diff
@@ -45,40 +50,46 @@ module.exports = {
...
     var plugins = registry.load('template');
 
-    return plugins.filter(function(plugin) {
+    return plugins.filter(function (plugin) {
       return plugin.name === 'ember-cli-htmlbars';
     })[0];
   },
 
-  _monkeyPatch_EmberDeprecate: function(htmlbarsCompilerPreprocessor) {
+  _monkeyPatch_EmberDeprecate: function (htmlbarsCompilerPreprocessor) {
```
Comment (rwjblue) [inline index.js:67]:
```suggestion
  _monkeyPatch_EmberDeprecate(htmlbarsCompilerPreprocessor) {
```

File: index.js [hunk 10]
```diff
@@ -115,19 +125,21 @@ module.exports = {
     };
   },
 
-  setupPreprocessorRegistry: function(type, registry) {
+  setupPreprocessorRegistry: function (type, registry) {
```
Comment (rwjblue) [inline index.js:128]:
```suggestion
  setupPreprocessorRegistry(type, registry) {
```

File: index.js [hunk 11]
```diff
@@ -115,19 +125,21 @@ module.exports = {
...
+      var htmlbarsCompilerPreprocessor = this._findHtmlbarsPreprocessor(
+        registry
+      );
 
       this._monkeyPatch_EmberDeprecate(htmlbarsCompilerPreprocessor);
     }
   },
 
-  lintTree: function(type, tree) {
+  lintTree: function (type, tree) {
```
Comment (rwjblue) [inline index.js:138]:
```suggestion
  lintTree(type, tree) {
```

File: tests/helpers/debug-test.js [hunk 12]
```diff
@@ -3,10 +3,10 @@ import { test } from 'qunit';
 
 let canRunDebugFunctions = false;
 
-runInDebug(() => canRunDebugFunctions = true);
+runInDebug(() => (canRunDebugFunctions = true));
```
Comment (rwjblue) [inline tests/helpers/debug-test.js:6]: WE should refactor this to just use `import { DEBUG } from '@glimmer/env'`

**Review by rwjblue — CHANGES_REQUESTED** [review]

File: index.js (hunk 4 above)
Comment (wagenet) [reply index.js:6]: We should also modernize the JS itself. I can take a first pass in this PR though.

**Review by rwjblue — APPROVED** [review]
LGTM

**Pull request merged**